src/style.css

```diff
@@ -1,3 +1,134 @@
-body {
+html {
+  --blue: #00acdc;
+  --dark-blue: #060b35;
+  --gray: #616070;
+  --white: #fff;
+  --font-family: "Roboto", sans-serif;
+  color: var(--dark-blue);
+  font-family: var(--font-family);
+}
+
+* {
   margin: 0;
+  box-sizing: border-box;
+  font-family: var(--font-family);
+}
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: var(--white);
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
```

Comment:
this is already written in the * selector, so there is no need to duplicate it here
```suggestion
```